ui/packages/catalog-ui-search/src/main/webapp/component/builder/builder.view.js

```diff
@@ -35,6 +34,13 @@ const ajaxCall = $.get({
   availableTypes = response
 })
 
+const announceMissingDefinition = definition =>
+  announcement.announce({
+    title: 'Missing Metacard Definition',
```

Comment:
Do we use `metacard` anywhere in the ui? I thought that was an internal implementation detail.

Reply:
Talked with @millerw8 and changed the wording.